controller/controller.go

```diff
@@ -270,6 +300,19 @@ func Install(rootDir string, model *model.SystemInstall, options args.Args) erro
 		model.AddBundle(language.RequiredBundle)
 	}
 
+	if encryptedUsed {
+		model.AddBundle(storage.RequiredBundle1)
+		model.AddBundle(storage.RequiredBundle2)
+	}
+
+	msg := fmt.Sprintf("Writing mount files")
+	prg = progress.NewLoop(msg)
+	log.Info(msg)
+	if err = storage.GenerateTabFiles(rootDir, model.TargetMedias); err != nil {
```

Comment:
so we really need a fstab file then?

Reply:
Yes, we really need the /etc/fstab and /etc/crypttab if we have encrypted swap or a non-root encrypted partition. We also need just /etc/fstab if someone creates a non-standard partition; aka not /, boot, /home, or /srv

Reply:
ok